src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/JSON.txt

```diff
@@ -0,0 +1,79 @@
+// (UTC-08:00) Pacific Time (US & Canada)
+#SETUP: TimeZoneInfo("Pacific Standard Time")
+#SETUP: OptionSetTestSetup
+
+>> JSON(Blank())
+"null"
+
+>> JSON("hello")
```

Comment:
More interesting string tests (for characters that need to be escaped according to the JSON spec):

    >> JSON("Back\slash")
    """Back\\slash"""

    >> JSON("Quotes "" and ' may be escaped")
    """Quotes \"" and ' may be escaped"""

    >> JSON($"Other {Char(13)} escaped {Char(10)} chars {Char(4)} should {Char(9)} also {Char(8)} be {Char(1)} escaped {Char(20)} properly")
    """Other \r escaped \n chars \u0004 should \t also \b be \u0001 escaped \u0014 properly"""